clients/cache/redis.go

```diff
@@ -0,0 +1,76 @@
+package cache
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/kava-labs/kava-proxy-service/logging"
+	"github.com/redis/go-redis/v9"
+)
+
+// RedisCache is an implementation of Cache that uses Redis as the caching backend.
+type RedisCache struct {
+	client *redis.Client
+	logger *logging.ServiceLogger
+}
+
+var _ Cache = (*RedisCache)(nil)
+
+func NewRedisCache(
+	ctx context.Context,
+	logger *logging.ServiceLogger,
+	address string,
+	password string,
+	db int,
+) (*RedisCache, error) {
+	client := redis.NewClient(&redis.Options{
+		Addr:     address,
+		Password: password,
+		DB:       db,
+	})
+
+	// Check if we can connect to Redis
```

Comment:
please remove this check here, but do update the health check to include checking reachablity of the cache

https://github.com/Kava-Labs/kava-proxy-service/blob/main/service/handlers.go#L15

Reply:
done